content/advanced/330_servicemesh_using_appmesh/integrate_observability/container_insight.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Container Insights"
+date: 2020-01-27T08:30:11-07:00
+weight: 91
+draft: false
+---
+
+#### Container Insights
+
+CloudWatch Container Insights is a fully managed service that collects, aggregates, and summarizes Amazon EKS metrics and logs. The CloudWatch Container Insights dashboard gives you access to the following information:
+* CPU and memory utilization
+* Task and service counts
+* Read/write storage
+* Network Rx/Tx
+* Container instance counts for clusters, services, and tasks
+
+Go to console, go to Cloudwatch, and go to `Container Insights` and `Performance monitoring, you can see the EKS Cluster insight
```

Comment:
This could be a little more clear.  e.g. 'Go the the CloudWatch console and under Container Insights navigate to Performance monitoring. Here you will see monitoring data for your EKS cluster" or something like that

Reply:
updated